doc/developers/contributing.rst

```diff
@@ -762,6 +762,93 @@ To test code coverage, you need to install the `coverage
 
 3. Loop.
 
+Monitoring performance
+======================
+
+*This section is heavily inspired from the* `pandas documentation 
+<https://pandas.pydata.org/docs/development/contributing.html#running-the-performance-test-suite>`_.
+
+
+When proposing changes to the existing code base, it's important to make sure
+that they don't introduce performance regressions. Scikit-learn uses
+`asv benchmarks <https://github.com/airspeed-velocity/asv>`_ to monitor the
+performance of a selection of common estimators and functions. The benchmark
+suite can be found in the `scikit-learn/asv_benchmarks` directory.
+
+To use all features of asv, you will need either `conda` or `virtualenv`. For
+more details please check the `asv installation webpage
+<https://asv.readthedocs.io/en/latest/installing.html>`_.
+
+First of all you need to install the development version of asv::
+
+  pip install git+https://github.com/airspeed-velocity/asv
```

Comment:
Why the development version?

Reply:
> So I tried to run,
asv continuous -b LogisticRegression upstream/master HEAD
this branch, but I'm getting the following error

To not get this error :D
(avoid benchmarking abstract classes is a not released yet)